components/logins/ios/Logins/Errors/LoginStoreError.swift

```diff
@@ -42,6 +42,10 @@ public enum LoginsStoreError: Error {
     /// This error is emitted if a request to a sync server failed.
     case Network(message: String)
 
+    /// This error is emitted if a call to `interrupt()` is made to
+    /// abort some operation.
+    case interrupted(message: String)
```

Comment:
nit: all the other cases seem to have `CamelCase` names but this one is lowercase?

Reply:
uppercase for these is considered bad style, we should rename all of them tbh... I'll file a bug.